src/test/java/com/thirdcc/webapp/web/rest/DebtResourceIT.java

```diff
@@ -125,61 +152,178 @@ public static Event createEventEntity() {
             .venue(DEFAULT_EVENT_VENUE)
             .fee(DEFAULT_EVENT_FEE);
     }
-
+    
+    public static EventAttendee createEventAttendeeEntity() {
+        return new EventAttendee()
+                .eventId(DEFAULT_EVENT_ID)
+                .userId(DEFAULT_USER_ID)
```

Comment:
this one actually no need la
you will assign loginUser later

Reply:
Resolved by #e02ef4a